plugins/com.google.cloud.tools.eclipse.bugreport/src/com/google/cloud/tools/eclipse/bugreport/ui/BugReportCommandHandler.java

```diff
@@ -31,8 +31,20 @@
   private static final String BUG_REPORT_URL =
       "https://github.com/GoogleCloudPlatform/google-cloud-eclipse/issues/new";
 
+  //@formatter:off
+  // should be kept up to date with .github/ISSUE_TEMPLATE.md
   private static final String BODY_TEMPLATE =
-      "- Cloud Tools for Eclipse Version: {0}\n- OS: {1} {2}\n- Java Version: {3}\n\n";
+      "- Cloud Tools for Eclipse Version: {0}\n"
```

Comment:
Suggestion to upgrade to the latest CT4E? (FYI, markdown doesn't seem to work here.)